data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -54,3 +54,43 @@ def get_bq_json_for_survey_questions_response_json(
         ],
         "imported_timestamp": get_current_timestamp()
     }
+
+
+def get_survey_answer_details(access_token: str, survey_id: str) -> list:
+    headers = get_surveymonkey_api_headers(access_token)
+    response = requests.get(
+        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/responses/bulk',
+        headers=headers
+    )
+    reponse_json = response.json()
+    return reponse_json["data"]
+
+
+def get_bq_json_for_survey_answers_response_json(
+    survey_response_json: dict
+) -> dict:
+    return {
+        "survey_answer_id": survey_response_json["id"],
+        "survey_id": survey_response_json["survey_id"],
+        "response_status": survey_response_json["response_status"],
+        "total_time_spent": survey_response_json["total_time"],
+        "date_modified": survey_response_json["date_modified"],
```

Comment:
The naming convention that I tried to follow is to put the type at the end, e.g. `modified_date` if it is a date, or `modified_timestamp` if it is actually a timestamp.

Reply:
Yes, `modified_timestamp` would be better, I am changing it.